protocol/synthetix/subgraph/tests/handleCollateralConfigured.ts

```diff
@@ -8,25 +8,25 @@ export default function test(): void {
   // Needs to be here because of Closures
   const now = new Date(1668448739566).getTime();
   const newCollateralConfiguredEvent = createCollateralConfiguredEvent(
-    Address.fromString(address),
+    address,
     true,
-    BigInt.fromI32(200),
-    BigInt.fromI32(50),
-    BigInt.fromI32(90),
-    Bytes.fromByteArray(Bytes.fromBigInt(BigInt.fromI32(10))),
-    BigInt.fromI32(500),
+    200,
```

Comment:
Can we have variable names please as unnamed function arguments when they are longer than one are confusing